configs/README.md

```diff
@@ -1,5 +1,5 @@
 ## Configuration File
-Example configuration files are provided in this directory. To build and run the given examples see the instructions [here](https://github.com/NOAA-OWP/cfe/blob/master/INSTALL.md). A detailed description of the parameters for model configuration (i.e., initialize/setup) is provided below. The asterisk (*) denotes calibratable parameters.
+Example configuration files are provided in this directory. To build and run the given examples see the instructions [here](https://github.com/NOAA-OWP/cfe/blob/master/INSTALL.md). A detailed description of the parameters for model configuration (i.e., initialize/setup) is provided below. The asterisk (*) denotes calibratable parameters. The has (#) shows the parameters only used with the Nash Cascade-based runoff.
```

Comment:
```suggestion
Example configuration files are provided in this directory. To build and run the given examples see the instructions [here](https://github.com/NOAA-OWP/cfe/blob/master/INSTALL.md). A detailed description of the parameters for model configuration (i.e., initialize/setup) is provided below. 


**Key**:

<sup>*</sup>: denotes parameter is calibratable

<sup>1</sup>: denotes the parameters only used when `surface_runoff_scheme` is `NASH_CASCADE`.
```

This renders as:

Example configuration files are provided in this directory. To build and run the given examples see the instructions [here](https://github.com/NOAA-OWP/cfe/blob/master/INSTALL.md). A detailed description of the parameters for model configuration (i.e., initialize/setup) is provided below. 

**Key**:

<sup>*</sup>: denotes parameter is calibratable

<sup>1</sup>: denotes the parameters only used when `surface_runoff_scheme` is `NASH_CASCADE`.

----------

It might be easiest to digest if the Key section is moved after the parameter table.

Reply:
@ajkhattak, if you don't implement the change here from @aaraney, please change "has" to "hash".